builtin/logical/consul/backend_old_test.go

```diff
@@ -0,0 +1,471 @@
+package consul
+
```

Comment:
I'd really like to see these two test files combined and to run all the tests against both versions of vault with the appropriate expectations based on the version.